packages/tools/src/types/AnnotationTypes.ts

```diff
@@ -11,6 +11,8 @@ type Annotation = {
   isVisible?: boolean;
   /** Has annotation data been invalidated (e.g., as a result of mouse interactions) */
   invalidated?: boolean;
+  /** If the annotation is auto generated from other annotations*/
```

Comment:
How about:
```
/** 
 * If the contour is auto generated, eg via interpolation or a remote process
 */
```
 
That may or may not be done from other annotations, and although the other lines are in the wrong format, the docs
generator won't pick those up, so having it in the full format is (unfortunately) needed.